server/@types/express/index.d.ts

```diff
@@ -27,9 +27,16 @@ import type {
 } from 'inductionForms'
 import type { InductionDto } from 'inductionDto'
 import type { UserDetails } from '../../services/userService'
+import type { CreateGoalsForm } from '../../routes/createGoal/validators/GoalForm'
 
 export default {}
 
+declare module 'express-serve-static-core' {
+  interface Response {
+    redirectWithSuccess?(path: string, message: string): void
```

Comment:
While we're only showing the success banner in the one place currently including the success banner in the layout template then allowing it to be shown from any controller via the `redirectWithSuccess()` method helps enforce a consist pattern and removes the need for boilerplate session code.